docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
+
+    ```
+    api:
+      host: abcdefghij.execute-api.us-east-1.amazonaws.com
+      basepath: /api
+    region: us-east-1
+    ```
+   
+## Authenticating with the DCE System
+
+DCE uses AWS Cognito to manager authentication and authorization.
+
+1. Open the AWS Console in your DCE Master Account and Navigate to AWS Cognito by typing `Cognito` in the search bar
+
+    ![Cognito](./cognito.png)
+
+1. Select `Manage User Pools` and click on the dce user pool.
+
+    ![Manage user pools](./manageuserpools.png)
+
+1. Select `Users and groups`
 
-5. Type `dce system deploy` to deploy dce to the AWS account specied in the previous step. This will be your new "DCE Master Account"
+    ![Users and Groups](./usersandgroups.png)
+    
+1. Create a user
 
-6. Edit your dce config file with the host and base url from the api gateway that was just deployed to your master account. This can be found in the master account under `API Gateway > (The API with "dce" in the title) > Stages > "Invoke URL: https://<host>/<baseurl>"`.
+    ![Create User](./createuser.png)
 
-7. Prepare a second AWS account to be your first "DCE Child Account"
+1. Name the user and provide a temporary password. You may uncheck all of the boxes and leave the other fields blank. This user will not have admin priviliges.
+
+    ![Quick start user](./quickstartuser.png)
+    
+1. Create a second user to serve as a system admin. Follow the same steps as you did for creating the first user, but name this one something appropriate for their role as an administrator.
+   
+    ![Quick start admin](./quickstartadmin.png)
+
+1. Create a group
+
+    ![Create group](./creategroup.png)
+
+1. Users in this group will be granted admin access to DCE. The group name must contain the term `Admin`. Choose a name and click on the `Create group` button.
+
+    ![Group name](./groupname.png)
+    
+1. Add your admin user to the admin group to grant them admin privileges. 
+    ![Quick start admin detail](./quickstartadmindetail.png)
+    ![Add to group](./addtogroup.png)
+
+1. Type `dce auth` in your command terminal. This will open a browser with a log in screen. Enter the username and password for the non-admin user that you created. Reset the password as prompted.
+
+    ![Quick start user login](./quickstartuserlogin.png)
+
+1. Upon successfully logging in, you will be redirected to a credentials page containing a temporary authentication code. Click the button to copy the auth code to your clipboard.
+   
+    ![Credentials Page](./credspage.png)
+    
+1. Return to your command terminal and paste the auth code into the prompt, then press enter.
+    
+    ```
+    dce auth
+    ✔ Enter API Token: : █ 
```

Comment:
@joshmarsh what's going on with pasting into the terminal? Is it doing this for you, too?


![image](https://user-images.githubusercontent.com/1153371/70548896-f3c34280-1b38-11ea-96d0-aac4c2a06d10.png)

Reply:
Open issue: https://github.com/Optum/dce-cli/issues/43